composer.json

```diff
@@ -32,16 +35,16 @@
         "php": "^7.3 || ~8.0.0",
         "laminas-api-tools/api-tools": "^1.3",
         "laminas-api-tools/api-tools-rest": "^1.3.2",
-        "laminas/laminas-hydrator": "^2.4.2 || ^3.0",
+        "laminas/laminas-hydrator": "^2.4.2 || ^3.0 || ^4.0",
         "laminas/laminas-stdlib": "^3.3",
         "laminas/laminas-view": "^2.11.3",
         "laminas/laminas-zendframework-bridge": "^1.0",
-        "phpro/zf-doctrine-hydration-module": "^2.0.1 || ^3.0 || ^4.1"
+        "api-skeletons/doctrine-orm-hydration-module": "^1.0"
     },
     "require-dev": {
-        "doctrine/doctrine-module": "^1.2.0 || ^2.1.7",
+        "doctrine/doctrine-module": "^4.0 || ^5.1",
         "doctrine/doctrine-mongo-odm-module": "^0.11.0 || ^1.0.0 || ^2.0.0",
-        "doctrine/doctrine-orm-module": "^1.1.8 || ^2.1.3",
+        "doctrine/doctrine-orm-module": "^4.0 || ^5.1",
```

Comment:
We should test only `^5.1`, and declare that `<5.1` is in `conflict:` (because we no longer test it)